myRental/scripts/dumpMysql.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+echo "Dumping MySQL schema into databaseSchema.sql"
+mysqldump -u shane -p --no-data --databases MY_RENTAL > ../databaseSchema.sql
```

Comment:
Will this work on other people's machines? If it does, can you change the username from 'shane' to something more anonymous. Also, how does the password get applied?